app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def posts_from_relation(relation, no_tests=true, with_pagination=true)
     posts = posts.paginate(page: page, per_page: 25) if with_pagination
     posts = posts.no_tests if no_tests
 
-    if (with_pagination && posts.total_pages <= 1) || posts.count <= 25
+    if (with_pagination && posts.total_pages <= 1) || posts.count(:all) <= 25
```

Comment:
Otherwise it calls `COUNT(posts.*, board.name AS board_name, …)` which triggers a syntax error. See https://github.com/rails/rails/pull/10710